tests/integration/api_v3_taxonomy_suggestions.t

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/perl -w
+
+use ProductOpener::PerlStandards;
+
+use Test::More;
+use ProductOpener::APITest qw/:all/;
+use ProductOpener::Test qw/:all/;
+use ProductOpener::TestDefaults qw/:all/;
+
+use File::Basename "dirname";
+
+use Storable qw(dclone);
+
+remove_all_users();
+
+remove_all_products();
+
+wait_application_ready();
+
+# Note: expected results are stored in json files, see execute_api_tests
+my $tests_ref = [
+	{
+		test_case => 'no-tagtype',
+		method => 'GET',
+		path => '/api/v3/taxonomy_suggestions',
+	},
+	{
+		test_case => 'incorrect-tagtype',
+		method => 'GET',
+		path => '/api/v3/taxonomy_suggestions?tagtype=not_a_taxonomy',
+	},
+	{
+		test_case => 'categories-no-string',
+		method => 'GET',
+		path => '/api/v3/taxonomy_suggestions?tagtype=categories',
+		expected_status_code => 200,
+	},
+	{
+		test_case => 'categories-string-strawberry',
+		method => 'GET',
+		path => '/api/v3/taxonomy_suggestions?tagtype=categories&string=strawberry',
+		expected_status_code => 200,
+	},
+	{
+		test_case => 'categories-term-strawberry',
+		method => 'GET',
+		path => '/api/v3/taxonomy_suggestions?tagtype=categories&term=strawberry',
+		expected_status_code => 200,
+	},
```

Comment:
So do we still support term ? Then it must be documented.

Reply:
It's an alias, I'm adding it to the doc.